bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/IPreferenceConstants.java

```diff
@@ -325,4 +325,15 @@ public interface IPreferenceConstants {
 	 */
 	String SHOW_KEYS_TIME_TO_CLOSE = "showCommandKeys_timeToClose"; //$NON-NLS-1$
 
+	/**
```

Comment:
`IPreferenceConstants` is internal interface, we can't use it from JDT/Xtext.

`org.eclipse.ui.IWorkbenchPreferenceConstants` is  better one.

Reply:
Wasn't the idea that we are adding a preference that was only internally visible? Or you mean `IPreferenceConstants` is not exported via its package?

Reply:
> Wasn't the idea that we are adding a preference that was only internally visible?

The idea was that we don't provide a **UI** for that preference (yet).

Reply:
> Or you mean IPreferenceConstants is not exported via its package?

This is also true, at least it wasn't accessible in my [draft PR](https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/548) using this preference.